apps/frontend/src/store/server.ts

```diff
@@ -118,6 +129,21 @@ class Server extends VuexModule implements IServerState {
     return axios.post('/users', userInfo);
   }
 
+  @Action({rawError: true})
+  public async updateUserInfo(userInfo: {
```

Comment:
the Params to the method should be defined in an interface (i.e. UpdateUserInterface)